include/core/UnwrapRef.h

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+#ifndef INCLUDE_CORE_UNWRAPREF_H_
+#define INCLUDE_CORE_UNWRAPREF_H_
```

Comment:
nit: following the pattern of all our other guards this would be `INCLUDED_ml_core_UnwrapRef_h`

(Note especially `INCLUDED` ending with `D`.)

Reply:
done